### PR TITLE
add setup for deploy and test

### PR DESCRIPTION
**Pull request by dubzn**
In this pull request, the necessary additions are made to the deploy script to:

- Deploy 2 ERC20 tokens (TYAS0, TYAS1).
- Create a pool with these two tokens and initialize it with values (FeeAmount.MEDIUM).
- Fund the account with both tokens to enable trading on YAS.
- Implementation of a wrapper for the mint() and swap() functions with a usage example in the script.

:warning:  Also in this pull request, there is a fix for the swap() function of YASPoool.

Closes: #122

### CODE REVIEW

**Review by dpinones — APPROVED** [review]
Great work!

File: src/contracts/yas_router.cairo [hunk 1]
```diff
@@ -1,6 +1,6 @@
 use starknet::ContractAddress;
 
-use yas::numbers::signed_integer::{i32::i32, i256::i256};
+use yas::numbers::signed_integer::{i32::i32, i256::i256, integer_trait::IntegerTrait};
```
[the remark this answers is not in the document]
Comment (dubzn) [reply src/contracts/yas_router.cairo:3]: done :heavy_check_mark:

**Comment by rcatalan98** [comment]
@dubzn could you link this PR to the appropriate issue?

**Review by rcatalan98 — APPROVED** [review]
Great job!

**Pull request merged**